workout-tracker/src/App.js

```diff
@@ -54,7 +53,7 @@ class App extends Component {
         <Switch>
           <Route exact path="/" component={LandingPage} />
           <Route path="/about" component={About} />
-          <Route exact path="/contact" component={ContactPage} />
+          <Route path="/about" component={LandingPage} />
```

Comment:
This is a duplicate route to line 55 and a switch component will route to the first path. This can be removed.